index.js

```diff
@@ -0,0 +1,52 @@
+document.getElementById("text").value = "";
+document.getElementById("text").addEventListener("keypress", function (e) {
+  if (e.key === "Enter") {
+    btnSubmit();
+  }
+});
+
+const data = [
+  {
+    text: "abc",
+    isChecked: false,
+  },
+  {
+    text: "abc2",
+    isChecked: true,
+  },
+];
+displayData(data);
+
+function btnSubmit() {
+  let txtValue = document.getElementById("text").value;
+  if (txtValue) {
+    // data.push(txtValue);
+    let task = {
+      isChecked: false,
+      text: txtValue,
+    };
+
+    data.push(task);
+  }
+  console.log(data);
+  displayData(data);
+}
+
+function displayData(data) {
+  document.getElementById("list").innerHTML = "";
+  if (data != "") {
+    for (let items of data) {
+      if (items.isChecked) {
```

Comment:
Do we  need this condition?
`checked` attribute on `input` accepts true and false value, you can try directly passing this value to the checked attribute.